vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -138,7 +137,6 @@ $global:map = @{
         "https://acs-mirror.azureedge.net/azure-cni/v1.4.7/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.4.7.zip"
     );
     "c:\akse-cache\calico\" = @(
-        "https://acs-mirror.azureedge.net/calico-node/v3.18.1/binaries/calico-windows-v3.18.1.zip",
         "https://acs-mirror.azureedge.net/calico-node/v3.19.0/binaries/calico-windows-v3.19.0.zip"
```

Comment:
we are going to support calico v3.20.0? https://github.com/Azure/AgentBaker/pull/1012

Reply:
Good catch. I am not aware of it. Let me add the cache at first.